### ISSUE TITLE
Bluetooth status is always TRUE in ConnectivityStatusChange event

### ISSUE THREAD

**Issue by baryshok** [nearit/react-native-connectivity-status#1]
### Environment

  Environment:
    OS: macOS Sierra 10.12.6
    Node: 9.2.1
    Yarn: 1.3.2
    npm: 5.5.1
    Watchman:  4.7.0
    Xcode:  Xcode 8.3.3 Build version 8E3004b
    Android Studio:  2.3 AI-162.4069837

  Packages: (wanted => installed)
    react: 16.0.0 => 16.0.0
    react-native: 0.50.3 => 0.50.3

  Target Platforms:
    Android: 5.0 (API 21)
    iOS: 8.0

### Steps to Reproduce

1. Subcsribe to connectivity status change events:
```
this._connectivityStatusChangeSubscription =
      ConnectivityManager.addStatusListener(this._handleConnectivityStatusChange);

. . .

_handleConnectivityStatusChange({ eventType, status }) {
  console.log(eventType, status);
}
```
2. Turn ON | OFF Bluetooth on a device and check status values in console.

### Expected Behavior

Status value should toggle between TRUE and FALSE when we turning ON | OFF Bluetooth.

### Actual Behavior

Status value is always TRUE regardless of the actual Bluetooth state.

<img width="583" alt="screen shot 2018-01-19 at 2 12 09 am" src="https://user-images.githubusercontent.com/16780712/35126390-473a412a-fcbe-11e7-8565-4a1b975ed25f.png">

**Comment by baryshok** [comment]
Hello @panz3r 
I've checked versions 1.1.3 and 1.2.0 and in both versions it worked the same way. Please, test it and tell if you can reproduce the issue. May be there's something wrong with my Environment.
Thanks!

**Comment by panz3r** [comment]
Hi @baryshok ,
Thanks for your feedback, I looked into your issue and I found the cause.

I just published a new version (`1.2.1`) containing the fix.

Soon I'll also release version `1.3.0` containing more fixes and enhancements.
